apps/web/src/studio/components/workflows/step-editor/WorkflowStepEditorInputsPanel.tsx

```diff
@@ -53,6 +55,9 @@ export const WorkflowStepEditorInputsPanel: FC<IWorkflowStepEditorInputsPanelPro
                         size={'sm'}
                         Icon={IconOutlineSave}
                         onClick={() => {
+                          segment.track('Step controls saved', {
+                            step: step?.type,
+                          });
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Should we call it channel or channel type? I think thats the convention in other places‏

Reply:
I talked with Radek about this one as it is specified in the issue, his response was:
"To me a channel is a high-level bucket that we support. here, we're in a specific context of a workflow, and our workflows have steps - and steps have types. yes, they are connected to channels, but they are not channels per se."